### PR TITLE
Publish recipes for ngram transitions work

### PR DESCRIPTION
**Pull request by vineelpratap**
Consider this as V0. 
TODO
- Test all the recipes again 
- More details to README. Possibly share ltr, WP based tokens, lexion, transition graph directly via S3 ...

### CODE REVIEW

File: recipes/diff_wfst/iamdb/1kwp_ngram2_optblank_prune0.cfg [hunk 1]
```diff
@@ -0,0 +1,68 @@
+{
```
Comment (awni) [inline recipes/diff_wfst/iamdb/1kwp_ngram2_optblank_prune0.cfg:1]: I would use `.json` extension. It's a bit odd to use `.cfg` for a JSON file..

File: recipes/diff_wfst/iamdb/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Scripts for preparing the IAMDB dataset can be found at `gtn_applications/datasets/iamdb.py` and preparing the wordpieces can be found at `gtn_applications/datasets/make_wordpieces.py`. 
+
+For preparing the pruned ngram transition graph, use `gtn_applications/scripts/build_transitions.py`.
```
Comment (awni) [inline recipes/diff_wfst/iamdb/README.md:3]: Use relative links for these files?

File: recipes/diff_wfst/iamdb/README.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+Scripts for preparing the IAMDB dataset can be found at `gtn_applications/datasets/iamdb.py` and preparing the wordpieces can be found at `gtn_applications/datasets/make_wordpieces.py`. 
+
+For preparing the pruned ngram transition graph, use `gtn_applications/scripts/build_transitions.py`. 
+
+
```
Comment (awni) [inline recipes/diff_wfst/iamdb/README.md:5]: Maybe we should add tables to this file which are organized according to the figures/tables in the paper and list the relevant jsons for each?

**Review by awni — APPROVED** [review]
Minor nits, thanks for adding these!

File: recipes/diff_wfst/iamdb/README.md (hunk 3 above)
Comment (vineelpratap) [reply recipes/diff_wfst/iamdb/README.md:5]: Let me create a new PR updating README

**Pull request merged**